subsys/bluetooth/host/hci_core.c

```diff
@@ -670,6 +674,30 @@ static int hci_le_read_remote_features(struct bt_conn *conn)
 	return 0;
 }
 
+#if defined(CONFIG_BT_BLUENRG_ACI)
+static int hci_send_aci_config_data_controller_mode(void)
+{
+	u8_t *data;
+
+	struct net_buf *buf;
+
+	buf = bt_hci_cmd_create(0xFC0C, sizeof(uint8_t)*(3));
```

Comment:
Please create a #define for the 0xfc0c, constructed with BT_OP(BT_OGF_VS, 0x0003). Also, `sizeof(uint8_t)*(3)` seems like a really complicated way of saying just `3` :)  (not to mention that we use the Zephyr int types instead of stdint)

Reply:
An even better option than just '3' would be if you create a parameter struct consisting of three descriptively named u8_t member variables. Then you can do something like:

```
struct vendor_param {
        u8_t foo;
        u8_t bar;
        u8_t baz;
};
...
static int hci_send_aci_config_data_controller_mode(void)
{
        struct vendor_param *param;

        buf = bt_hci_cmd_create(VENDOR_OP, sizeof(*param));
        ...
        param->foo = 0x2c;
        param->bar = 0x01;
        param->baz = 0x01;
        ...
```

Even better if you also create defines for the magic values `0x2c`, `0x01` and `0x01`